src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -7566,7 +7582,55 @@ internal static void InvalidateCache()
             }
         }
 
-        #endregion
```

Comment:
This shouldn't be removed -- it corresponds to `#region Runtime helpers` at line 7418. The following 2 methods should be within that region.